user/advanced-topics/guivm.md

```diff
@@ -0,0 +1,134 @@
+
+> Note: The setup is done using `SaltStack` formulas with the `qubesctl` tool. When executing it, apply step can take time because it needs to download latest Fedora XFCE TemplateVM and install desktop dependencies.
+
+
+### Hybrid GuiVM `sys-gui`
+
+Here, we describe how to setup `sys-gui` that we call *hybrid mode* or referenced as a *compromised solution* in [GUI Domain](https://www.qubes-os.org/news/2020/03/18/gui-domain/#the-compromise-solution).
+
+[![sys-gui](/attachment/posts/guivm-hybrid.png)](/attachment/posts/guivm-hybrid.png)
+
+In `dom0`, enable the formula for `sys-gui` with pillar data:
+
+```bash
+sudo qubesctl top.enable qvm.sys-gui
+sudo qubesctl top.enable qvm.sys-gui pillar=True
+```
+
+then, execute it:
+
+```bash
+sudo qubesctl --all state.highstate
+```
+
+You can now disable the `sys-gui` formula:
+```bash
+sudo qubesctl top.disable qvm.sys-gui
+```
+
+At this point, you need to shutdown all your running qubes as the `default_guivm` qubes global property has been set to `sys-gui`. In order to use `sys-gui` as GuiVM, you need to logout and select `lightdm` session to *Gui Domain (sys-gui)*. Once logged, you are running `sys-gui` as fullscreen window and you can perform any operation as if you would be in `dom0` desktop.
+
+> Note: In order to go back to `dom0` desktop, you need to logout and then, select `lightdm` session to *Session Xfce*.
+
+### GPU GuiVM `sys-gui-gpu`
+
+Here, we describe how to setup `sys-gui-gpu` which is a `GuiVM` with *GPU passthrough* in [GUI Domain](https://www.qubes-os.org/news/2020/03/18/gui-domain/#gpu-passthrough-the-perfect-world-desktop-solution).
+
+[![sys-gui-gpu](/attachment/posts/guivm-gpu.png)](/attachment/posts/guivm-gpu.png)
+
+In `dom0`, enable the formula for `sys-gui-gpu` with pillar data:
+
+```bash
+sudo qubesctl top.enable qvm.sys-gui-gpu
+sudo qubesctl top.enable qvm.sys-gui-gpu pillar=True
+```
+
+then, execute it:
+
+```bash
+sudo qubesctl --all state.highstate
+```
+
+You can now disable the `sys-gui-gpu` formula:
+
+```bash
+sudo qubesctl top.disable qvm.sys-gui-gpu
+```
+
+At this point, you need to reboot your Qubes OS machine in order to boot into `sys-gui-gpu`.
+
+> None: For some platforms, it can be sufficient to shutdown all the running qubes and starting `sys-gui-gpu`. Unfortunately, it has been observed that detaching and attaching some GPU cards from `dom0` to `sys-gui-gpu` can freeze computer. We encourage reboot to prevent any data loss.
+
+Once, `lightdm` is started, you can log as `user` where `user` refers to the first `dom0` user in `qubes` group and with corresponding `dom0` password.
```

Comment:
The user password situation is weird. Until solved, please add a link to https://github.com/QubesOS/qubes-issues/issues/6740